src/products/views/ProductCreate/ProductCreate.tsx

```diff
@@ -282,6 +288,16 @@ export const ProductCreateView: React.FC<ProductCreateProps> = ({ params }) => {
     updateChannelsOpts.loading ||
     updateVariantChannelsOpts.loading;
 
+  const channelsErrors = [
+    ...(updateVariantChannelsOpts.data?.productVariantChannelListingUpdate
+      ?.errors || []),
+    ...(updateChannelsOpts.data?.productChannelListingUpdate?.errors || []),
+  ];
+  const errors = [
+    ...(productCreateOpts.data?.productCreate.errors || []),
+    ...(productVariantCreateOpts.data?.productVariantCreate.errors || []),
+  ];
+
```

Comment:
can you split this into functions that will extract your errors by a given "opts"? Perhaps it can be useful for later (a utility?) eg. a function that will take an array of opts and return for you merged errors of each entry

Reply:
Sure, I've found out that we have `getMutationErrors` which I can use here